src/main/java/appeng/tile/storage/TileChest.java

```diff
@@ -538,15 +543,17 @@ public void channelRender( final MENetworkChannelsChanged c )
 	}
 
 	@Override
-	public IMEMonitor getItemInventory()
-	{
-		return this.itemCell;
-	}
-
-	@Override
-	public IMEMonitor getFluidInventory()
+	public <T extends IAEStack<T>, C extends IStorageChannel<T>> IMEMonitor<T> getInventory( IStorageChannel<T> channel )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') C is not used in the method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2326)